src/analyze/State.re

```diff
@@ -174,6 +174,41 @@ let refmtForUri = (uri, package) =>
     }
   };
 
+let fmtCmdForUri = (~formatWidth, ~interface, uri, package) => {
```

Comment:
It looks like this can have a return type of `result(string, string)`, not `result(option(string), string)` -- could you change that?